app/views/works/_required_creators_table.html.erb

```diff
@@ -23,26 +23,38 @@
         <th>&nbsp;</th> <!-- delete icon-->
       </tr>
     </thead>
-    <tbody class="sortable"></tbody>
+    <tbody class="sortable">
+      <% creators = if @work.nil? || @work.resource.creators.empty? 
+                      [nil]
```

Comment:
I would have expected this to be `[]` rather that `[nil]`. Am I reading it wrong?